### ISSUE TITLE
Blog improvements

### ISSUE THREAD

**Issue by sublayerio** [armada-alliance/armada-alliance#144]
- Add support for time to read (e.g. 3 mins read)
- Add subtle progress bar when scrolling through lengthy posts
- Improve content readability so that it's more comforting to read.
- Add "next article" feature with ability to go to that article by just continuing to scroll

**Comment by AstroWa3l** [comment]
Would be nice to make the font a bit darker and larger.

**Comment by sublayerio** [comment]
@AstroWa3l You're speaking of light or dark mode?

**Comment by AstroWa3l** [comment]
@sublayerio I notice the font could be darker on "light" mode, the dark mode could be a little bit "bolder" maybe a tiny bit darker but it is not the issue.

**Comment by AstroWa3l** [comment]
Another improvement could be to add a "editor" role, so if one member spends time editing the blog post they are also awarded as a contributor.

**Comment by sublayerio** [comment]
@AstroWa3l Good point. I've added documentation for it to: https://github.com/armada-alliance/armada-alliance/blob/main/services/website/content/README.md

Right now all identities tied to the post are displayed on the page. I propose hiding the people with editor role for the time being to keep the list of identities lightweight. At a later stage we can display the editors of the post in a subtle way as to give them credit. But most importantly, the posts editors have edited can now count to their contributions.

**Comment by AstroWa3l** [comment]
Sounds good to me :)
